1-exercises/A-undefined/exercise.js

```diff
@@ -12,11 +12,13 @@
 // Example 1
 let a;
 console.log(a);
+function has no parimeter
```

Comment:
That's not quite right I'm afraid, here undefined happens beacause the var called a doesn't yet have a value